src/latexify/ast_utils_test.py

```diff
@@ -38,15 +38,15 @@ def test_make_attribute() -> None:
 @pytest.mark.parametrize(
```

Comment:
Ditto: as this test case only runs on older versions than 3.8, we can simply remove it.

Reply:
I’m not completely sure I understand this part. Could you help me clarify or commit to update? 
I noticed there were some test cases added earlier, but I’m unsure of their purpose, so I didn’t want to remove them just yet. I’ve updated the code to stop using the deprecated functions based on warnings from the latest version, but I wanted to double-check with you first.

Reply:
I removed this test case